collections-csharp/UpdateItemInListCSharp/src/UpdateItemInListCSharp/Models/Book.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace UpdateItemInListCSharp.Models
+{
+    public class Book(string author, string isbn, bool isCheckedOut, string title)
+    {
+        public string Author { get; set; } = author;
+
+        public string ISBN { get; set; } = isbn;
+
+        public bool IsCheckedOut { get; set; } = isCheckedOut;
+
+        public string Title { get; set; } = title;
```

Comment:
Better to have Title as the first property of book, I feel it will add more readability

Reply:
Should I only adjust the parameters in the constructor or restructure the order of the properties?
Personally, I always have the properties sorted alphabetically so that I work in a standardised way and don't always have to think about it.

Reply:
I prefer to have most important to least important properties in the class. I am expecting both, to re-arrange properties and order of the parameters.